homeassistant/components/switch/acer_projector.py

```diff
@@ -114,7 +114,7 @@ def _write_read_format(self, msg):
         match = re.search(r'\r(.+)\r', awns)
         if match:
             return match.group(1)
-        return STATE_UNKNOWN
+        return None
```

Comment:
This method is also used to populate state attributes, so this will be a breaking change.